### ISSUE TITLE
Sourmash threads option incorrect

### ISSUE THREAD

**Issue by conmeehan** [jodyphelan/malaria-profiler#23]
Hi,

When I try running malaria-profiler, I get the following error log, which I think is pointing to an issue with passing threads to sourmash. It doesn't run past this so I cannot type my genomes at all unfortuately. Any idea how to fix?

Thanks,
Conor

# malaria_profiler error report

* OS: linux
* malaria-profiler version: 0.0.10
* pathogen-profiler version: 5.2.1
* Program call:
```
{'no_cleanup': False, 'logging': 'INFO', 'read1': None, 'read2': None, 'bam': None, 'fasta': 'GCA_000002415.2_ASM241v2_genomic.fna', 'vcf': None, 'platform': 'illumina', 'resistance_db': None, 'external_resistance_db': None, 'species_db': 'malaria-db', 'external_species_db': None, 'mitochondria_seq_name': None, 'prefix': 'GCA_000002415.2_ASM241v2', 'dir': '.', 'csv': False, 'txt': True, 'pdf': False, 'add_columns': None, 'add_mutation_metadata': False, 'call_whole_genome': False, 'depth': '0,0', 'af': '0.1,0.4', 'strand': '0,0', 'sv_depth': '0,10', 'sv_af': '0.5,0.9', 'sv_len': '100000,50000', 'min_species_relative_abundance': 2.0, 'mapper': 'bwa', 'caller': 'freebayes-haplotype', 'taxonomic_software': 'sourmash', 'kmer_counter': 'kmc', 'calling_params': None, 'coverage_tool': 'bedtools', 'species_only': False, 'no_trim': False, 'no_coverage_qc': False, 'no_samclip': True, 'run_delly': False, 'no_species': False, 'add_variant_annotations': False, 'threads': 1, 'ram': 2, 'max_barcode_missing': 0.1, 'supplementary_bam': None, 'db_dir': '/mnt/storage/conda/cmeehan/malaria-profiler/share/malaria_profiler', 'verbose': 0, 'delly_vcf': None, 'func': <function main_profile at 0x7f724f466b90>, 'software_name': 'malaria_profiler', 'version': '0.0.10', 'tmp_prefix': '8644e266-d52c-4686-9f50-ca0f263d1f18', 'files_prefix': './8644e266-d52c-4686-9f50-ca0f263d1f18', 'samclip': False, 'coverage_qc': True, 'data_source': 'fasta', 'no_delly': True, 'species_conf': {'db-schema-version': '1.0.0', 'version': {'db-schema-version': '1.0.0', 'name': 'malaria-db', 'repo': 'https://github.com/pathogen-profiler/malaria-db.git', 'branch': 'main', 'commit': '847e3c08', 'status': 'clean', 'author': 'jodyphelan', 'date': 'Sun Aug 10 11:07:21 2025 +0100'}, 'files': {'variables': 'variables.json', 'sourmash_db': 'plasmodium.sbt.zip', 'accessions': 'species_info.csv'}, 'variables': {'db-schema-version': '1.0.0', 'version': {'db-schema-version': '1.0.0', 'name': 'malaria-db', 'repo': 'https://github.com/pathogen-profiler/malaria-db.git', 'branch': 'main', 'commit': '847e3c08', 'status': 'clean', 'author': 'jodyphelan', 'date': 'Sun Aug 10 11:07:21 2025 +0100'}, 'files': {'variables': 'variables.json', 'sourmash_db': 'plasmodium.sbt.zip', 'accessions': 'species_info.csv'}}, 'sourmash_db': '/mnt/storage/conda/cmeehan/malaria-profiler/share/malaria_profiler/malaria-db/plasmodium.sbt.zip', 'accessions': '/mnt/storage/conda/cmeehan/malaria-profiler/share/malaria_profiler/malaria-db/species_info.csv'}}
```
## Traceback:
```
  File "/mnt/storage/conda/cmeehan/malaria-profiler/bin/malaria-profiler", line 471, in <module>
    args.func(args)
  File "/mnt/storage/conda/cmeehan/malaria-profiler/bin/malaria-profiler", line 117, in main_profile
    species_prediction = mp.get_species(args)
  File "/mnt/storage/conda/cmeehan/malaria-profiler/lib/python3.10/site-packages/malaria_profiler/utils.py", line 49, in get_species
    return pp.get_species_prediction(args)
  File "/mnt/storage/conda/cmeehan/malaria-profiler/lib/python3.10/site-packages/pathogenprofiler/cli.py", line 475, in get_species_prediction
    species_hits = get_taxonomic_hits(args)
  File "/mnt/storage/conda/cmeehan/malaria-profiler/lib/python3.10/site-packages/pathogenprofiler/cli.py", line 364, in get_taxonomic_hits
    hits = sketch.get_species_hits(
  File "/mnt/storage/conda/cmeehan/malaria-profiler/lib/python3.10/site-packages/pathogenprofiler/taxonomy.py", line 64, in get_species_hits
    sequence_hits = self.classify(ref_db, *args, **kwargs)
```
## Value:
```
SourmashSig.classify() got an unexpected keyword argument 'threads'```
